architecture-building-systems/CityEnergyAnalyst
# remove faulty input requests

the PR resolves issue #2144
to test in Dashboard:
1. run create new project (zone, district, terrain, streets)
2. run data helper
the script should run through, and a `technology` folder should be created inside the `inputs` folder
